.github/workflows/container.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           context: .
           platforms: linux/amd64
-          file: Containerfile
+          file: apps/web/Containerfile
```

Comment:
The `Containerfile` moved, but the build "content" did not: the context remains the repo root.